### ISSUE TITLE
Tristramleecher - followers dont pickit if not attacking - enhancement needed

### ISSUE THREAD

**Issue by DarkHorseDre** [kolton/d2bot-with-kolbot#1074]
**Scenario**:  
followers (using autobuild) set to use tristram leecher, follower entry script.
followers have attacks disabled as they often get caught attacking and lose the leader/die!

leader is running lead entry and tristram (plus 2 other) scripts with walkclear and leech portal enabled.

**Issue**: 
followers don't pickit - they walk past pickit items

**Proof**:
if I enable any attack, they will then pickit after attacking, even if attack fails/mob not killed.

**Desired outcome**: 
tristramleecher also pickits regardless of attack

I realise this may be the case for other scripts, but tristram leecher behaves strangely even in town - bot maintains tight control of interface and movement even when waiting for tp (cant move followers, cant open inv. or char stats etc).  
on a run, this tight control may be due to the need to follow leader. but if an item is in close range (e.g. <=20) surely they can just pick?

leechers also tend to exit game or stall and exit mid trist run..

**Comment by 5noop** [comment]
Well if you removed `Attack.Clear` then yes the leechers won't pick items anymore, it's normal behavior. Now if you want them to pick items but not attacking, then you have to replace `Attack.clear(10);` by `Pickit.pickItems();`.

**Comment by DarkHorseDre** [comment]
no I mean I set their attacks to: 
Config.AttackSkill		= [-1, -1, -1, -1, -1];

Going by what you said, I could customise to only pickit instead of attack in the tristramleech.js?
(I'd rather not do that as I may use it properly in future, which is why I'd rather they were able to pick regardless of attack - main issue was they were not picking gold or pots so ran out in hc!)

**Comment by braxIRL** [comment]
In your pickit file do you have pots and gold lines in there? Autoequip .nip file doesn't have pots or gold in there unless you add it in. Also most pickits don't have minor healing pots, minor mana pots, etc. I'm sure if you edit your pickits that it should pick up gold and pots :)

**Comment by DarkHorseDre** [comment]
@braxIRL yeah all works as usual - problem is the tristram leech script - it behaves differently from other scripts (I'm not using autoequip). check the workaround mentioned in this thread.
